[2021-12-22 17:32:05,302] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.face_crop scheduled__2021-12-20T00:21:00+00:00 [queued]>
[2021-12-22 17:32:05,307] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.face_crop scheduled__2021-12-20T00:21:00+00:00 [queued]>
[2021-12-22 17:32:05,307] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:32:05,307] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-22 17:32:05,307] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:32:05,316] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): face_crop> on 2021-12-20 00:21:00+00:00
[2021-12-22 17:32:05,318] {standard_task_runner.py:52} INFO - Started process 25991 to run task
[2021-12-22 17:32:05,320] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline1', 'face_crop', 'scheduled__2021-12-20T00:21:00+00:00', '--job-id', '126', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmpd4lhwq3t', '--error-file', '/tmp/tmpc1lv31l7']
[2021-12-22 17:32:05,320] {standard_task_runner.py:77} INFO - Job 126: Subtask face_crop
[2021-12-22 17:32:05,337] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline1.face_crop scheduled__2021-12-20T00:21:00+00:00 [running]> on host myamya
[2021-12-22 17:32:05,377] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline1
AIRFLOW_CTX_TASK_ID=face_crop
AIRFLOW_CTX_EXECUTION_DATE=2021-12-20T00:21:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-20T00:21:00+00:00
[2021-12-22 17:32:05,378] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-22 17:32:05,378] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker run -it -v /home/myamya/project/image_files/:/home/myamya/project/image_files/ google_api:0.8 /bin/bash init.sh ']
[2021-12-22 17:32:05,385] {subprocess.py:85} INFO - Output:
[2021-12-22 17:32:06,191] {subprocess.py:89} INFO - 0
[2021-12-22 17:32:06,191] {subprocess.py:89} INFO - 2
[2021-12-22 17:32:06,191] {subprocess.py:89} INFO - 3
[2021-12-22 17:32:06,191] {subprocess.py:89} INFO - @@ /home/myamya/project/image_files/images/2264.jpg
[2021-12-22 17:32:07,200] {subprocess.py:89} INFO - [Info] /home/myamya/project/image_files/images/2264.jpg: Find face from in position [31, 39, 621, 725] and skew angle [0.83626616, 1.2407098, -0.2966359]
[2021-12-22 17:32:07,546] {subprocess.py:89} INFO - [Info]  /home/myamya/project/image_files/images/2264.jpg: Crop face [31, 39, 621, 725] and write it to file : /home/myamya/project/image_files/cropped_image/2264.jpg
[2021-12-22 17:32:07,546] {subprocess.py:89} INFO - training 1
[2021-12-22 17:32:07,546] {subprocess.py:89} INFO - @@ /home/myamya/project/image_files/images/2266.jpg
[2021-12-22 17:32:08,171] {subprocess.py:89} INFO - [Info] /home/myamya/project/image_files/images/2266.jpg: Find face from in position [31, 39, 621, 725] and skew angle [0.83626616, 1.2407098, -0.2966359]
[2021-12-22 17:32:08,491] {subprocess.py:89} INFO - [Error] /home/myamya/project/image_files/images/2266.jpg: Duplicated image
[2021-12-22 17:32:08,492] {subprocess.py:89} INFO - @@ /home/myamya/project/image_files/images/2263.jpeg
[2021-12-22 17:32:08,953] {subprocess.py:89} INFO - [Error] /home/myamya/project/image_files/images/2263.jpeg: cannot process file : 'y'
[2021-12-22 17:32:08,954] {subprocess.py:89} INFO - @@ /home/myamya/project/image_files/images/2265.jpeg
[2021-12-22 17:32:09,414] {subprocess.py:89} INFO - [Error] /home/myamya/project/image_files/images/2265.jpeg: cannot process file : 'y'
[2021-12-22 17:32:09,414] {subprocess.py:89} INFO - ## label 1 has 1 of training data
[2021-12-22 17:32:09,414] {subprocess.py:89} INFO - number of datas per label  [1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
[2021-12-22 17:32:09,643] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-22 17:32:09,673] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline1, task_id=face_crop, execution_date=20211220T002100, start_date=20211222T083205, end_date=20211222T083209
[2021-12-22 17:32:09,710] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-22 17:32:09,737] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
